code/mob/living/silicon/ai/holograms.dm

```diff
@@ -0,0 +1,71 @@
+//AI HOLOGRAMS
+/mob/living/silicon/ai/proc/create_hologram()
+	if (!deployed_to_eyecam)
+		boutput(src, "Deploy to an AI Eye first to create a hologram.")
+		return
+	if (islist(holograms) && holograms.len >= max_holograms)
```

Comment:
I think you can get away with `if (length(holograms) >= max_holograms)`

Reply:
Should work, and be slightly faster.